### PR TITLE
Add doxygen

### LINKED ISSUE #9734: Add Doxygen
### Brand Name

Doxygen

### Website

doxygen.nl

### Popularity Metric

The similarweb rank is 474,701. See https://www.similarweb.com/website/doxygen.nl

### Official Resources for Icon and Color

SVG: https://github.com/doxygen/doxygen/blob/master/templates/html/doxygen.svg
Color: `#2c4aa8` from website header

### Additional Comments

The favicon would have suited more as the brand icon for a badge but I could not find an svg for that.

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
doxygen

**Issue:** closes #9734 

**Similarweb rank:** 474,701. See https://www.similarweb.com/website/doxygen.nl
**GitHub Stars:** 5k. See https://github.com/doxygen/doxygen/tree/master

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from https://github.com/doxygen/doxygen/blob/2e73e6a53abe00a57b7eb0ea5ed6474baf4ebab7/templates/html/doxygen.svg
Color: `#2c4aa8` taken from website header www.doxygen.nl

### CODE REVIEW

File: sdk.mjs [hunk 1]
```diff
@@ -119,7 +119,7 @@ export const htmlFriendlyToTitle = (htmlFriendlyTitle) =>
     .replace(/&#([0-9]+);/g, (_, num) => String.fromCodePoint(parseInt(num)))
     .replace(
       /&(quot|amp|lt|gt);/g,
-      (_, ref) => ({ quot: '"', amp: '&', lt: '<', gt: '>' })[ref],
+      (_, ref) => ({ quot: '"', amp: '&', lt: '<', gt: '>' }[ref]),
```
[the remark this answers is not in the document]
Comment (PratyushJayachandran) [reply sdk.mjs]: I tried several times. The linter is putting it.

File: icons/doxygen.svg
Comment (adamrusted) [inline icons/doxygen.svg:1]: I'm seeing some differences on this one, @PratyushJayachandran - would you mind taking another look?
image

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Review by PeterShaggyNoble — APPROVED** [review]
I'm not seeing those differences, @adamrusted, everything lines up perfectly. Could you have another look before merging, though, in case I'm wrong?

**Review by service-paradis — APPROVED** [review]
LGTM 🎉

**Pull request merged**